src/WordPress.php

```diff
@@ -510,6 +517,16 @@ private function configure_site_files() {
 		$custom_conf_source      = SITE_WP_TEMPLATE_ROOT . '/config/nginx/user.conf.mustache';
 		$process_user            = posix_getpwuid( posix_geteuid() );
 
+		if ( ! empty( $this->site_data['site_aliases'] ) ) {
+
+			$domains = explode( ',', $this->site_data['site_aliases'] );
```

Comment:
explode only `if ( 'subdom' === $this->site_data['app_sub_type'] )`, otherwise directly use the input. No need for extra processing of expoling and then rejoining for no reason.